build/attestations/index.md

```diff
@@ -49,6 +49,16 @@ You can opt in to add either the SBOM or provenance attestation type, or both.
 $ docker buildx build --sbom=true --provenance=true .
 ```
 
+> SBOM generation is not enabled by default. It must be opted in using `--sbom=true` option.
+>
+> However, provenance generation is enabled by default with `mode=min,inline-only=true` option.
+>
+> It can be disabled by setting `--provenance=false` or using `BUILDX_NO_DEFAULT_ATTESTATIONS` environment variable.
+>
+> If explicitly enabled using `--provenance=true`, provenance attestation is created with `mode=max` option by default.
+>
+> See [Provenance attestation](./slsa-provenance.md) section for more details.
```

Comment:
Thanks for this contribution!
Just some formatting suggestions.

```suggestion
> **Note**
>
> Provenance attestations are enabled by default, with the `mode=min` option.
> You can disable provenance attestations using the `--provenance=false` flag,
> or by setting the `BUILDX_NO_DEFAULT_ATTESTATIONS` environment variable:
>
> ```console
> $ export BUILDX_NO_DEFAULT_ATTESTATION=1
> ```
>
> Using the `--provenance=true` flag attaches provenance attestations with `mode=max`
> by default. See [Provenance attestation](./slsa-provenance.md) for more details.
```

Reply:
@dvdksn I don't think we have a common place for `BUILDX_` vars atm i nthe docs right? Might be worth to have smth like https://docs.docker.com/engine/reference/builder/#buildkit-built-in-build-args in our docs too?